src/coreComponents/linearAlgebra/interfaces/hypre/mgrStrategies/HypreMGRCompositionalMultiphaseFVM.hpp

```diff
@@ -0,0 +1,128 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file HypreMGRCompositionalMultiphaseFVM.hpp
```

Comment:
Personally I would be in favor of naming the file just `CompositionalMultiphaseFVM.hpp` (same for others), to be consistent with class name. I don't think we have any issues with having repeated file names in the project, and the path prefix of `hypre/mgrStrategies` already contains the relevant scoping information. Not insisting on this though, just a thought.

Reply:
Agreed, done.